### DISCUSSION TITLE
Columns vs layout divs?

### DISCUSSION THREAD

**Discussion by kjohnsen** [quarto-dev/quarto-cli#2750] [Q&A] [answered]
I see in the [guide's section on figures](https://quarto.org/docs/authoring/figures.html#figure-panels), a `{layout-ncols=x}` or `{layout="..."}` approach is taught, where child items are placed accordingly. However, in the [section on revealjs](https://quarto.org/docs/presentations/revealjs/#multiple-columns), a `columns` div with `column` children approach is taught. What are the pros and cons of each? The layout approach seems to be more flexible...why wouldn't I use it in slides as well as document figures?

**Comment by mcanouil** [comment, marked as answer]
You can use both and for me it does not make much difference.
the main differences between the two approaches are: one is more of a "grid"-like approach while the other can only split the are in several columns.

I w s thinking of making a PR to Quarto documentation website to add the other approaches in the "multiple columns" section of several formats, including the one you quoted.
